docs/developer-docs/latest/update-migration-guides/migration-guides/v4/migration-guide-4.5.1-to-4.6.1.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Migrate from 4.5.1 to 4.6.1 - Strapi Developer Docs
+description: Learn how you can migrate your Strapi application from 4.5.1 to 4.6.1.
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/update-migration-guides/migration-guides/v4/migration-guide-4.5.1-to-4.6.1.html
+---
+
+# v4.5.1 to v4.6.1 migration guide
+
+The Strapi v4.5.1 to v4.6.1 migration guide upgrades v4.5.1 to v4.6.1. We introduced a configuration for webhooks to receive populated relations. The migration guide consists of:
+
+- Upgrading the application dependencies
+- Changing the webhooks configuration (optional)
+- Reinitializing the application
+
+## Upgrading the application dependencies to 4.6.1
+
+:::prerequisites
+Stop the server before starting the upgrade.
+:::
+
+1. Upgrade all of the Strapi packages in `package.json` to `4.6.1`:
+
+   ```json
+   // path: package.json
+
+   {
+     // ...
+     "dependencies": {
+       "@strapi/strapi": "4.6.1",
+       "@strapi/plugin-users-permissions": "4.6.1",
+       "@strapi/plugin-i18n": "4.6.1"
+       // ...
+     }
+   }
+   ```
+
+2. Save the edited `package.json` file.
+
+3. Run either `yarn` or `npm install` to install the new version.
+
+::: tip
+If the operation doesn't work, try removing your `yarn.lock` or `package-lock.json`. If that doesn't help, remove the `node_modules` folder as well and try again.
+:::
+
+## Changing the webhooks configuration (optional)
+
+By default, and for backwards compatibility reasons, webhooks will receive the populated relations. We do recommend to disable this behavior if you don't need it.
```

Comment:
```suggestion
By default, and for backward compatibility reasons, webhooks will receive the entity with its relations populated again. We do recommend to disable this behavior if you were not using it, as it may cause performance issues when having many relations. If you need populated relations in your webhook, we recommend doing a separate query in your webhook listenner to fetch the entity only with the necessary data.
```

Reply:
It is something we can put in the doc also ? Like a hint or else